src/python/pants/core/goals/run_test.py

```diff
@@ -70,11 +71,13 @@ def create_mock_run_debug_adapter_request(
 
 class TestRunFieldSet(RunFieldSet):
     required_fields = ()
+    run_in_sandbox_behavior = RunInSandboxBehavior.NOT_SUPPORTED
```

Comment:
Issue?

Reply:
Oh huh I guess this only needed for this one type specifically in the run test...

Yeah ok I was thinking all plugin tests would need it, but this PR is proof otherwise.